lte/gateway/c/core/oai/test/spgw_task/state_creators.cpp

```diff
@@ -61,8 +61,8 @@ s_plus_p_gw_eps_bearer_context_information_t* make_bearer_context(imsi64_t imsi,
   IMSI64_TO_STRING(imsi, (char*)(&sgw->imsi.digit), IMSI_BCD_DIGITS_MAX);
   sgw->imsi_unauthenticated_indicator = 20;
   strncpy(pgw->msisdn, msisdn, strlen(msisdn));
-  sgw->mme_teid_S11 = 300;
-  sgw->s_gw_teid_S11_S4 = 400;
+  sgw->mme_teid_S11 = 300 + teid;
```

Comment:
Please add the purpose of this fix in the PR summary.

Reply:
Sure I will explain in summary also
In test case, TestUEContextConversion, it creates multiple bearers. So each bearer must be identified with unique teid. But here teid was hard coded